internal/proxy/proxy.go

```diff
@@ -223,6 +223,12 @@ func (p *Proxy) handleJenkinsUIRequest(w http.ResponseWriter, r *http.Request, r
 
 		//Break the process if the Jenkins is idled, set a cookie and redirect to self
 		if isIdle {
+			err = p.idler.UnIdle(ns)
```

Comment:
I guess this is correct. If we cannot make the unidle call, no need to set the cookie, right?